### PR TITLE
[WIP] Update 0.3.10

### PR DESCRIPTION
**Pull request by ChrisAlderson**

### CODE REVIEW

**Comment by VitorVRS** [comment]
Audio Lang feature its already done?

**Comment by ChrisAlderson** [comment]
No not yet, but this will switch out 2 `butter-provider` modules, one for the movies and one for the anime. This way all three content types are being provided by the `popcorn-api`.

**Comment by Chriz007** [comment]
Does any of this fix the crashing many people have been experiencing while watching something? It happens randomly

**Comment by pbassut** [comment]
@Chriz007 Just tested it: it doesn't fix it.

File: src/app/lib/providers/trakttv.js [hunk 1]
```diff
@@ -515,6 +516,25 @@
...
+                AdvSettings.set('traktTokenTTL', auth.expires_in);
+                self.authenticated = true;
+                App.vent.trigger('system:traktAuthenticated');
+                defer.resolve(true);
+            }).catch(function (err) {
+                AdvSettings.set('traktToken', '');
+                AdvSettings.set('traktTokenTTL', '');
+                AdvSettings.set('traktTokenRefresh', '');
+                defer.reject(err);
+>>>>>>> a383973... nw.js => 0.13 migration due to architecture Changes
```
Comment (VitorVRS) [inline src/app/lib/providers/trakttv.js]: Merge conflicts here.

**Comment by GuiChevis** [comment]
Something is wrong with the Butter player, all episodes I play i get this error, seems to work in VLC.

image

**Comment by ChrisAlderson** [comment]
@guichevis We are still working on compiling NWJS for Windows, Mac and Linux are working.

**Pull request merged**